modules/nrf_wifi/bus/rpu_hw_if.c

```diff
@@ -483,10 +483,12 @@ int rpu_enable(void)
 		goto rpu_pwroff;
 	}
 
+#if 0
```

Comment:
local change?

Reply:
We cannot have `rpu_validate_comms()` anymore since it will not work unless firmware is loaded and for a independent bus lib, this wont work. I think its best we move this function as we will have bustest now, plan to do in the next commit

Reply:
OK, I have put this under #ifdef CONFIG_WIFI_NRF70 - that should take care of the issue while not changing functionality for the core wifi driver.